### DISCUSSION TITLE
starlette 0.14.2 issue with fastAPI due to deprecation of UJSONResponse

### DISCUSSION THREAD

**Discussion by L1ghtn1ng** [fastapi/fastapi#6766] [Questions] [answered]
From me having a look at what is importable in starlette it needs to use JSONResponse not UJSONResponse as that seems to have been removed
![image](https://user-images.githubusercontent.com/3628354/113640388-3a647680-9673-11eb-8e88-a0ec72c407f6.png)

Would be good to get this fixed ASAP and a new release pushed out to fix this issue as need to use this version of starlette and due to fastapi not having support for this version of starlette is being a release blocker for me

**Comment by Kludex** [comment, marked as answer]
https://github.com/tiangolo/fastapi/issues/2342
https://github.com/tiangolo/fastapi/issues/2493
https://github.com/tiangolo/fastapi/pull/2335

**Comment by L1ghtn1ng** [comment]
Doh missed these, can we go about increasing its priority to get this resolved?

**Comment by Kludex** [comment]
This will probably be addressed by the end of the month.

But, as I said before on one of those issues: just don't pin Starlette. FastAPI already installs the right Starlette version.

Btw, as it is a duplicated issue, do you mind closing it?

**Comment by L1ghtn1ng** [comment]
with the version that fastapi installs unfortunately has missing imports that are in newer versions that I need and yeah sure will close it

**Comment by tiangolo** [comment]
Thanks for the help here @Kludex ! 👏  🙇 

Thanks for reporting back and closing the issue @L1ghtn1ng  👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
